### PR TITLE
Add controls for Kubernetes v1.20 with CIS v1.0.0 and extra check

### PR DESCRIPTION
**Pull request by bigdatasourav**

### CODE REVIEW

**Review by rajlearner17 — COMMENTED** [review]
@bigdatasourav pls associate test results with it.

**Comment by bigdatasourav** [comment]
[test.txt](https://github.com/turbot/steampipe-mod-kubernetes-compliance/files/7513853/test.txt)

**Review by rajlearner17 — APPROVED** [review]
LGTM

File: cis_kubernetes_v120/cis_kubernetes_v120.sp [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    plugin             = "kubernetes"
+  }
+}
+
+benchmark "cis_kubernetes_v120" {
+  title         = "CIS Kubernetes v1.20"
+  description   = "CIS Kubernetes v1.20 benchmark provides prescriptive guidance for establishing a secure configuration posture for Kubernetes 1.19 - 1.20."
+  tags          = local.cis_kubernetes_v120_common_tags
+  children = [
+    benchmark.cis_v100,
```
Comment (misraved) [inline cis_kubernetes_v120/cis_kubernetes_v120.sp:14]:
```suggestion
    benchmark.cis_v100    
```

File: query/job/job_container_privilege_port_mapped.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  -- Required Columns
+  c ->> 'name' as resource,
+  case
+    When p ->> 'name' is null then 'skip'
+    when cast(p ->> 'containerPort' as integer) <= 1024 then 'alarm'
+    else 'ok'
+  end as status,
+  case
+    When p ->> 'name' is null then 'No port mapped.'
+    when cast(p ->> 'containerPort' as integer) <= 1024 then p ->> 'name'|| ' mapped with privileged port.'
```
Comment (misraved) [inline query/job/job_container_privilege_port_mapped.sql:11]:
```suggestion
    when cast(p ->> 'containerPort' as integer) <= 1024 then p ->> 'name'|| ' mapped with a privileged port.'
```

**Review by misraved — COMMENTED** [review]
1. Make changes as per the review comments
2. Update the reason section of the queries as per the suggested pattern

File: query/deployment/deployment_hostipc_sharing_disabled.sql [hunk 3]
```diff
@@ -0,0 +1,17 @@
+select
+  -- Required Columns
+  uid as resource,
+  case
+    when template -> 'spec' ->> 'hostIPC' = 'true' then 'alarm'
+    else 'ok'
+  end as status,
+  case
+    when template -> 'spec' ->> 'hostIPC' = 'true' then 'Deployment pods share host IPC namespaces.'
+    else 'Deployment pods cannot share host IPC namespaces.'
```
[the remark this answers is not in the document]
Comment (bigdatasourav) [reply query/deployment/deployment_hostipc_sharing_disabled.sql:10]: cannot, existing in cisa

**Pull request merged**